packages/react-dom/src/events/accumulateTwoPhaseListeners.js

```diff
@@ -76,11 +82,13 @@ export default function accumulateTwoPhaseListeners(
     // usual two phase accumulation using the React fiber tree to pick up
     // all relevant useEvent and on* prop events.
     let node = event._targetInst;
+    let lastHostComponent;
 
     // Accumulate all instances and listeners via the target -> root path.
     while (node !== null) {
-      // We only care for listeners that are on HostComponents (i.e. <div>)
+      // Handle listeners that are on HostComponents (i.e. <div>)
       if (node.tag === HostComponent) {
+        lastHostComponent = node.stateNode;
         // For useEvent listenrs
         if (enableUseEventAPI && accumulateUseEventListeners) {
           // useEvent event listeners
```

Comment:
Hmm actually the bug seems to be here. When the flag is on, we follow this code path, and call getListenersFromTarget on the state node. But for some reason it may be null at this point (unmounted?). I don’t understand why this is possible but it seems to happen from the trace.